### PR TITLE
correct description of front channel SLO

### PR DESCRIPTION
**Pull request by dodok1**
There should be emphasized state of implementation on various cas clients.

### CODE REVIEW

File: docs/cas-server-documentation/installation/Logout-Single-Signout.md [hunk 1]
```diff
@@ -83,6 +72,25 @@ A sample SLO message:
 </samlp:LogoutRequest>
 ```
 
+### Front Channel
+
+CAS issues asynchronous AJAX `GET` logout requests via `JSONP` to authenticated services.
+The expected behaviour of the CAS client is to invalidate the application web session. 
+
+Support for front channel logout calls may be available only in development version of CAS clients.
```
Comment (mmoayyed) [inline docs/cas-server-documentation/installation/Logout-Single-Signout.md]: Please take out this line (#80)

Comment (mmoayyed) [reply docs/cas-server-documentation/installation/Logout-Single-Signout.md]: Or put in a different way: Front channel logout may not be available for all CAS clients. Ensure your CAS client does support this behavior before trying it out.

**Comment by dodok1** [comment]
Changed to proposed wording - anyway, is there way to check implementation?

**Comment by mmoayyed** [comment]
Other than downloading the client and trying it out, I don't know of a different way to check for support.

**Pull request merged**